nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-ldap/src/main/java/org/nuxeo/ecm/directory/ldap/LDAPSession.java

```diff
@@ -145,10 +145,13 @@ public DirContext getContext() {
 
     @Override
     protected DocumentModel createEntryWithoutReferences(Map<String, Object> fieldMap) {
+        // Make a copy of fieldMap to avoid modifying it
+        Map<String, Object> fieldMapCopy = new HashMap<>(fieldMap);
```

Comment:
You can simply say `fieldMap = new HashMap<>(fieldMap)`. This just puts the result into `fieldMap` instead of creating a new local variable `fieldMapCopy` (we don't have a code style prohibition on never changing passed-in parameters) so the diff will also be minimized.

Reply:
ok 👍